src/index.scss

```diff
@@ -30,346 +37,268 @@ $mobile-width: 750px;
 	@return m#{a}x(#{$numbers});
 }
 
-a:hover {
-	text-decoration: none;
-}
-body {
-	margin: 0;
-	font-family: Poppins, sans-serif;
-}
-html, body, #root, #app, .full-height {
-	height: 100%;
-}
-
 .desktop-hide {
 	@include desktop {
 		display: none;
 	}
 }
 .mobile-hide {
-	@include tablet {
+	@include mobile {
 		display: none;
 	}
 }
-
 .flex-container {
```

Comment:
Where is this being used?

Reply:
`flex-container` is currently used in the input bar, it used to also be used in the nav-bar but I've changed it after some refactoring. I find it to be a useful combination of attributes so I think it's worth keeping for now since I see myself using it for the schedule.